demo/src/App.js

```diff
@@ -172,13 +195,45 @@ function App() {
           </Button>
           <Button
             disabled={!isPyramid || isLoading}
-            onClick={toggleOverview}
+            onClick={toggleLinkedView}
             variant="outlined"
             size="small"
             fullWidth
           >
-            {overviewOn ? 'Hide' : 'Show'} Picture-In-Picture
+            {useLinkedView ? 'Overview' : 'Linked'}
           </Button>
+          {!useLinkedView ? (
+            <Button
+              disabled={!isPyramid || isLoading}
+              onClick={toggleOverview}
+              variant="outlined"
+              size="small"
+              fullWidth
+            >
+              {overviewOn ? 'Hide' : 'Show'} Picture-In-Picture
+            </Button>
+          ) : (
+            <>
+              <Button
+                disabled={!isPyramid || isLoading}
+                onClick={toggleZoomLock}
+                variant="outlined"
+                size="small"
+                fullWidth
+              >
+                {zoomLock ? 'Unlock' : 'Lock'} Zoom
+              </Button>
+              <Button
+                disabled={!isPyramid || isLoading}
+                onClick={togglePanLock}
+                variant="outlined"
+                size="small"
+                fullWidth
+              >
+                {panLock ? 'Unlock' : 'Lock'} Pan
+              </Button>
```

Comment:
I actually liked the no-padding.  I had noticed it as well but left it because it gives a bit of distinction between toggling demos and then controlling within the demo you are on.